test_app/schema/schema.yaml

```diff
@@ -348,3 +348,43 @@ classes:
     annotations:
       mutable: false
       plural: ImmutableTypes
+
+  AutoUpdatedType:
+    is_a: Entity
+    mixins:
+      - EntityMixin
+    attributes:
+      name:
+        description: This is a field description
+        range: string
+        required: true
+      auto_inc_field:
+        range: integer
+        annotations:
+          auto_increment: true
+      default_value:
+        range: string
+        ifabsent: "ifabsent_value"
+      default_sa_func:
+        range: date
+        annotations:
+          default_sa_function: now()
+      default_callable:
+        range: date
+        annotations:
+          default_value_callable: datetime.datetime.now
+      onupdate_callable:
+        range: date
+        annotations:
+          onupdate: datetime.datetime.now
+      onupdate_sa_func:
+        range: date
+        annotations:
+          onupdate_sa_function: now()
+      combo_field:
+        range: date
+        annotations:
+          default_sa_function: now()
```

Comment:
the behavior here would be that the default function is called on create and the update function is called on all updates?